Automate/Speckle.Automate.Sdk/Runner.cs

```diff
@@ -1,9 +1,9 @@
 using System.CommandLine;
 using System.Diagnostics.CodeAnalysis;
+using Newtonsoft.Json.Schema;
 using Newtonsoft.Json.Schema.Generation;
 using Newtonsoft.Json.Serialization;
```

Comment:
Ping @JR-Morgan as I'm fairly certain we should not be using Newtonsoft.Json and instead use our own `Speckle.Newtonsoft.Json`

This may be an oversight that precedes this PR though, judging by the other 2 lines